### PR TITLE
(bitdefender-usb-immunizer) Changed to manual updating

### PR DESCRIPTION
**Pull request by AdmiringWorm**
Updated it and moved it to manual directory (no update since 2011)

Requires [PUSH bitdefender-usb-immunizer] on merge.

### CODE REVIEW

File: manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+    <version>2.0.1.901</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/bitdefender-usb-immunizer</packageSourceUrl>
     <owners>chocolatey</owners>
+    <title>Bitdefender USB Immunizer</title>
     <authors>Bitdefender SRL</authors>
-    <licenseUrl>http://labs.bitdefender.com/projects/usb-immunizer/overview/</licenseUrl>
     <projectUrl>http://labs.bitdefender.com/projects/usb-immunizer/overview/</projectUrl>
-    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/edba4a5849ff756e767cba86641bea97ff5721fe/icons/bitdefender-usb-immunizer.png</iconUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/9282e769979fbb235869051249a70d56517f4ff9/icons/bitdefender-usb-immunizer.png</iconUrl>
+    <copyright>What is the copyright</copyright>
```
Comment (pascalberger) [inline manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec]: Why "What is the copyright"? Shouldn't the tag be omitted if copyright is unclear?

File: manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec [hunk 2]
```diff
@@ -19,12 +23,12 @@ For years, Autorun-based malware has been atop of the worldwide e-threat landsca
 Introduced back in the Windows XP era to facilitate software installations from CD-ROM media for non-technical computer users, the Autorun feature has rapidly become the infection vector of choice for cyber-criminals.
 
 Bitdefender USB Immunizer is a response to this growing issue.
+
+It disables autorun-related threats before they access the computer. Once installed, it constantly watches for newly inserted USB storage devices and immunizes them on the fly. If you accidentally plug in an infected USB drive that has not been immunized, the computer will not auto-execute the piece of malware located on the USB storage device.
+
```
Comment (pascalberger) [inline manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec]: Remove multiple line breaks (or does chocolatey take them into consideration?)

File: manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec (hunk 1 above)
Comment (AdmiringWorm) [inline manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec]: ops, forgot about that one. I have a script that updates the nuspec file. forgot to change this one.

File: manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec (hunk 2 above)
Comment (AdmiringWorm) [inline manual/bitdefender-usb-immunizer/bitdefender-usb-immunizer.nuspec]: Multiple line breaks means a new paragraph.

**Comment by AdmiringWorm** [comment]
@pascalberger the change on the icon is smaller size, I ran pngquant on it (part of my converting script)

**Pull request merged**